samples/display/ili9340/README.rst

```diff
@@ -51,11 +101,35 @@ Build this sample application with the following commands:
 
 See :ref:`nucleo_l476rg_board` on how to flash the build.
 
+For :ref:`nrf52_pca10040`, build this sample application with the following commands:
+
+.. zephyr-app-commands::
```

Comment:
@carlescufi Are you recommending we use ``:tool: all`` for all samples now?

Reply:
No, because `:tool: all` is the default now when `:tool:` is not specified. Later on I plan to make `:tool: west` the default, and only the doc that explicitly uses `:tool: all` will display instructions both for west and cmake

Reply:
Ah. OK.  I missed that you were changing the default tool from ``all`` to ``west``.  Thanks for clarifying.